<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>285</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans Hebrew</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="total_program" stretch="2,10,2,5">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_title">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans Hebrew</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Enter your date of birth</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="to_enter" stretch="1,5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="entry_titles">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_day">
           <property name="font">
            <font>
             <family>Noto Sans Hebrew</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Day: </string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_month">
           <property name="font">
            <font>
             <family>Noto Sans Hebrew</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Month: </string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_year">
           <property name="font">
            <font>
             <family>Noto Sans Hebrew</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Year: </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="entry_users">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="line_day">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_month">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_year">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_get_star_sign">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>411</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans Hebrew</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Get Star Sign</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_title_star_sig">
         <property name="font">
          <font>
           <family>Noto Sans Hebrew</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Your Star Sign: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_star_sign">
         <property name="font">
          <font>
           <family>Noto Sans Hebrew</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
